src/pages/pattern/pattern-page/pattern-page.js

```diff
@@ -26,7 +27,10 @@ const tableTitles = config.tableHeadRowTitles.patterns;
 
 const PatternPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed